### ISSUE TITLE
Can't type in blishHud modules

### ISSUE THREAD

**Issue by markodiuss** [blish-hud/Blish-HUD#866]
I can't type any text in any fields in blishHud modules. Tried to reinstall but it doesn't help me(

**Comment by dlamkins** [comment]
Do you have any anti-virus software?

We use both keyboard and mouse hooks to recieve input - which can be blocked by some 3rd party software.

**Comment by markodiuss** [comment]
Nope. One more thing i just found. I can type numbers, but not characters like A-z

**Comment by dlamkins** [comment]
Could you please tell me what sort of keyboard layout you use?

**Comment by markodiuss** [comment]
EN International and Ukr.

**Comment by entrhopi** [comment]
That's strange. I can't even confirm this with my wired setup.
Could you do me a favor and create a `Settings` folder in your Blish directory and try it again? This should place all configuration there instead of your user directory.

While we're at it ... 
are you running GW2 and/or Blish as administrator?
are you playing fullscreen or windowed fullscreen?

**Comment by markodiuss** [comment]
I tried to erun blish as administrator - didn't help.
I played with windowed fullscreen.

Thx. i will try to create Settings folder in Blish folder

UPD: Created Settings folder in Blish folder, and it doesn't help(

**Comment by markodiuss** [comment]
Maybe Blish have any instrument to debug?

**Comment by entrhopi** [comment]
Yes!
You can start Blish with the '--debug' parameter and attach the log file from the `logs` folder here. Maybe we'll find some hints in there.

Is keyboard input not working in all of Blish (i.e. API key input field) or are just installed modules affected?

**Comment by markodiuss** [comment]
Keyboard doesnt work in all Blish.
There may be a problem with the fonts. I mean, maybe some fonts used by Blish Hud are not available in my system, and that's why nothing is printed? Although I can enter numbers, I just don't have any ideas anymore...
I run it in debug mode, but it doesn't create log folder\or file. Im shure it was running in debyg because i saw message near checkbox "Disabled because Blish HUD was started with --debug specified"

**Comment by entrhopi** [comment]
Fonts are precompiled and shipped with the ZIP file you download from the Github repository. There is no need to have them installed on your system.

Depending on if you are using the 'regular' or 'local' (with a Settings folder created) version, you should find the log files either in `Documents\Guild Wars 2\addons\blishhud\logs` or `Settings\logs`. 

Please also have a look here: https://blishhud.com/docs/user/troubleshooting

**Comment by Nordggs** [comment]
The problem is in the Cyrillic alphabet.
When you use several keyboard layouts, for some reason the program cannot change the layout in the text input field. I had a problem with the keyboard shortcuts for changing layouts. It was Shift-Alt.
I changed the keyboard shortcuts to Ctrl-Shift. And it worked. I was able to type text in any text field in all modules.
(Sorry if I’m not expressing myself accurately. I’m writing with automatic translation)
